index.src.html

```diff
@@ -2355,13 +2367,38 @@
             |clone|.
     2. Return |clone|.
 
+: To check if a {{AudioDataInit}} is a
+    <dfn>valid AudioDataInit</dfn>, run these steps:
+:: 1. If {{AudioDataInit/sampleRate}} = `0`, return `false`.
+    2. If {{AudioDataInit/numberOfFrames}} = `0`, return `false`.
+    3. If {{AudioDataInit/numberOfChannels}} = `0`, return `false`.
+    4. Verify {{AudioDataInit/data}} has enough data by running the following
+        steps:
+        1. Let |totalSamples| be the product of multiplying
+            {{AudioDataInit/numberOfFrames}} by
+            {{AudioDataInit/numberOfChannels}}.
+        2. Let |bytesPerSample| be the number of bytes per sample, as defined by
+            the {{AudioDataInit/format}}.
+        3. Let |totalSize| be the product of multiplying |bytesPerSample| with
+            |totalSamples|.
+        4. Let |dataSize| be the size in bytes of {{AudioDataInit/data}}.
+        5. If |dataSize| is less than |totalSize|, return false.
+    5. Return `true`.
+
+<div class='note'>
+Note: We assume that {{AudioDataInit}}'s {{AudioDataInit/data}}'s memory layout
```

Comment:
I don't know that the plural first person is common in specs. Who would `we` be ?

Reply:
Removed.